pinot-common/src/main/java/org/apache/pinot/common/function/TransformFunctionType.java

```diff
@@ -65,6 +65,7 @@ public enum TransformFunctionType {
 
   AND("and"),
   OR("or"),
+  NOT("not"),
```

Comment:
let's add a comment indicating `NOT_EQUAL` and `NOT_IN` are not covered by this branch

Reply:
Added the comment.